tests/cli/test_deploy.py

```diff
@@ -2678,7 +2688,20 @@ async def test_deploy_without_name_in_prefect_yaml_interactive(
             invoke_and_assert,
             command="deploy --all",
             expected_code=0,
-            user_input="y" + readchar.key.ENTER + "test-name-2" + readchar.key.ENTER,
+            user_input=(
```

Comment:
This is my first time seeing these tests for the interactive prompts and they are :sun_with_face: